script.cinemavision/lib/main.py

```diff
@@ -886,24 +1044,21 @@ def save(self, as_new=False, export=False):
             if not path:
                 return
         else:
-            contentPath = kodiutil.getSetting('content.path')
+            contentPath = kodiutil.getPathSetting('content.path')
             if not contentPath:
                 xbmcgui.Dialog().ok(T(32503, 'No Content Path'), ' ', T(32553, 'Please set the content path in addon settings.'))
                 return
 
             path = cinemavision.util.pathJoin(contentPath, 'Sequences')
 
-        name = self.name
+        pathName = self.sequenceData.pathName
 
-        if not name or as_new or export:
-            name = xbmcgui.Dialog().input(T(32554, 'Enter name for file'), name)
-            if not name:
+        if not pathName or as_new or export:
+            pathName = xbmcgui.Dialog().input(T(32554, 'Enter name for file'), pathName)
```

Comment:
localize if possible

Reply:
This one already is, unless you mean `pathName` which isn't localizable.

Reply:
I mean "enter name for file"

Reply:
That's why it's wrapped in `T`: `T(32554, 'Enter name for file')`. The `T` function localizes, and the text is just there for reference. It isn't actually used in the function.